### ISSUE TITLE
Avast! blocks the setup as IDP.Generic

### ISSUE THREAD

**Issue by scara** [gurnec/HashCheck#31]
While trying to recover HashCheck due to Windows 10 being updated to Creators (#15) Avast! prevents me to be successful:

C:\Users\<username>\AppData\Local\Temp\NSU14F.TMP => IDP.Generic

Any hint?

TIA,
Matteo

**Comment by ghost** [comment]
It's most likely a false positive; I had VirusTotal analyze all of HashCheck's files, and none of the AV software flagged it. (That includes Avast.) Try either updating Avast if it's an older version, re-installing HashCheck, or temporarily disable Avast.

**Comment by ghost** [comment]
Hi,
me too i get the same thing 
C:\Users\<username>\AppData\Local\Temp\NSU14F.TMP => **IDP.Generic**
i am seriously aware of that ???

![avastui_2017-08-10_21-39-46](https://user-images.githubusercontent.com/30388530/29191707-87ed2fba-7e16-11e7-8254-76874ea5cc67.png)

**Comment by nastyagrifon** [comment]
That's false positive. Avast marks it "IDP.Generic" for suspicious behavior just like AVG Antivirus does.

You can ignore it or whitelist the application. Sometimes updating antivirus software helps as well

**Comment by joeswann** [comment]
This just happened with AVG - it broke Insomnia when it tried to update and then wouldn't let it reinstall.

**Comment by rootupdate** [comment]
This is usually a False Positive error So you can Ignore it but if you are Getting [IDP GENERIC](https://www.rootupdate.com/2019/06/what-idp-generic-avast.html) error even for Legitimate Apps then you need to scan files through virus total or try upgrading your antivirus

**Comment by margretfree** [comment]
Yup, false positive. Unless we decided to stuff the game with malware and I didn't get the memo :D

Lately, we had some issues with Norton as well. Antiviruses often mistake in-game optimizations for malware code.

if you have an option in your client to report [idp.generic](https://winerrorfixer.com/idp-generic/) false positive please do so as well.

**Comment by softwareinspect** [comment]
Have you ever encounter [IDP.Generic](https://softwareinspect.com/what-is-idp-generic-virus/) threat in your PC? Ever wonder what it is and how to stop it. Read this article to know What is IDP.Generic Virus and How to stop it?

**Comment by droidgyaan** [comment]
Most youtube faces the[ youtube comments not loading](http://droidgyaan.com/fix-youtube-comments-not-loading-in-chrome/) issue. Read this article to learn how to fix youtube comments not loading in chrome.

**Comment by windowspcsecret** [comment]
Would you like to root your machine with your PC? Here we share a Kingroot PC download guide. The most recent version has been shared by us directly download.
[kingroot for pc
](https://windowspcsecrets.com/kingroot-for-pc/)

**Comment by GanaAmrutha** [comment]
Scribd is a fantastic subscription service for e-books and audiobooks. It is a platform for sharing and viewing papers and presentations. Scribd contains articles and journals on a wide range of themes, including technology, science, business, literature, and many more. Furthermore, a person can browse and read the data in the document without registering. However, in order to swap a download for a file or document, the user must subscribe to Scribd's paid membership or submit specific documents. This Scribd book downloader guide will cover sites where you may use to download any journal, document, file, thesis, article, and so on from Scribd for free.
[scribd downloader](https://www.streamingtip.com/scribd-downloader/)
